functions.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Theme functions file.
+ *
+ * @package WPAMPTheme
+ */
+
+// Include required classes.
+require_once get_template_directory() . '/includes/class-amp-travel-blocks.php';
+
+
+if ( ! function_exists( 'travel_setup' ) ) :
```

Comment:
@miina,
Thanks for putting the blocks logic in a separate class.

Could you create another file, `includes/functions.php`, which has a function like:

```
function amp_travel_theme() {
    return AMP_Travel_Theme::get_instance();
}
```

Then, the file that this comment is attached to could simply call `amp_travel_theme()->init()` after requiring the files above. (This borrows heavily from @ThierryA's work on another project).

Then, the current logic inside `if ( ! function_exists( 'travel_setup' ) ) :` could move into a new `AMP_Travel_Theme` class: `includes/class-amp-travel-theme.php`. That logic could be inside a method that `AMP_Travel_Theme::init()` calls. 

It'd be great if you could also create a method `AMP_Travel_Theme::instantiate_classes()`, which would have the line `new AMP_Travel_Blocks();`

Reply:
@kienstra Thanks for the comments, the file structure issues should be addressed now, let me know if there's anything else that might need changing.

Reply:
Thanks, @miina. The [structure looks good](https://github.com/xwp/travel/pull/13/files#diff-f81f3dff36bf04b7a070d642c858e883R1).